ui-components/src/navigation/ProgressNav.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react"
-import { OnClientSide } from "../helpers/nextjs"
+import { OnClientSide } from "@bloom-housing/shared-helpers"
```

Comment:
So we can't make `shared-helpers` a dependency of `ui-components` because that will leak any Next-specific stuff in. Might need to walk this back…

Reply:
Refactored this component to remove the dependency, ty!